.github/workflows/release.yml

```diff
@@ -19,32 +19,25 @@ jobs:
     runs-on: ubuntu-latest
     environment: terraform-registry
     steps:
-      -
-        name: Checkout
+      - name: Checkout
         uses: actions/checkout@v2.4.0
-      -
-        name: Unshallow
+      - name: Unshallow
         run: git fetch --prune --unshallow
       - name: Install semantic-release Dependency
         run: npm install -g semantic-release conventional-changelog-conventionalcommits
-      -
-        name: Tag Release
+      - name: Tag Release
         run: semantic-release
-      -
-        name: Set up Go
+      - name: Set up Go
         uses: actions/setup-go@v2
         with:
           go-version: 1.17
-      -
-        name: Import GPG key
+      - name: Import GPG key
         id: import_gpg
-        uses: hashicorp/ghaction-import-gpg@v2.1.0
-        env:
-          # These secrets will need to be configured for the repository:
-          GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
-          PASSPHRASE: ${{ secrets.GPG_PRIVATE_KEY_PASSPHRASE }}
-      -
-        name: Run GoReleaser
+        uses: crazy-max/ghaction-import-gpg@v5.0.0
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.GPG_PRIVATE_KEY_PASSPHRASE }}
```

Comment:
formatter when a little crazy on this file, but this is the only section that actually changed